spec/presenters/tree_builder_archived_spec.rb

```diff
@@ -0,0 +1,103 @@
+describe TreeBuilderArchived do
+  context 'TreeBuilderArchived' do
+    before do
+      extend TreeBuilderArchived
```

Comment:
Another one of these extends in the before... The general pattern for testing a module is creating a temporary class that includes it, and test on that temporary class...

```ruby
let(:instance) { Class.new.tap { include TreeBuilderArchived }.new }
```